formats/protobuf/provider_test.go

```diff
@@ -1,72 +1,21 @@
 package protobuf_test
 
 import (
-	"bytes"
-	"encoding/hex"
-	"fmt"
 	"io/ioutil"
 	"log"
-	"math/rand"
-	"os/exec"
 	"path/filepath"
 	"testing"
 
 	"github.com/goto/stencil/formats/protobuf"
 	"github.com/stretchr/testify/assert"
 )
 
-func runProtoc(
```

Comment:
what is this? and why we deleted this?

Reply:
Looks like this is moved to different file formats/protobuf/testutils.go but files which is not _test will be part of the build. Should we add _test suffix to the next file?

Reply:
its a utility method which is used to run protoc command. Its being used at two places. Hence moved it to testutil inside protobuf as its a utility for protobuf. This is not a test file. Its a utils for test files.

Reply:
Can you move this utils file to a package called test_helper, so that we can mark the package to be excluded for the test coverage?

Reply:
It should be done in makefile. go test ./... if it is not there already, we should add it.